src/hotspot/share/runtime/globals.hpp

```diff
@@ -2077,8 +2077,8 @@ const intx ObjectAlignmentInBytes = 8;
   product(bool, UseArrayMarkWordCheck, NOT_LP64(false) LP64_ONLY(true),     \
           "Use bits in the mark word to check for flat/null-free arrays")   \
                                                                             \
-  product(ccstrlist, ForceNonTearable, "", DIAGNOSTIC,                      \
-          "List of inline classes which are forced to be atomic "           \
+  product(ccstrlist, ForceAtomicAccess, "", DIAGNOSTIC,                     \
+          "List of inline classes which are forced to be atomic access "    \
           "(whitespace and commas separate names, "                         \
           "and leading and trailing stars '*' are wildcards)")              \
                                                                             \
```

Comment:
Internally to the JVM, the term "atomic" is acceptable, as long as
in context it is correctly understood as *single-access* atomicity.
(Which it is.)  The product flag here is at the boundary between
the JVM internals and the end-user (who sets the JVM flag on
the command line).  At that point, it is better to use the external
term `NonTearable`, which (for pedagogical reasons, as explained
elsewhere) does not use the word "atomic" but rather a synonym.